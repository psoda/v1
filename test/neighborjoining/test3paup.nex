#NEXUS


begin data;
	dimensions ntax=4 nchar=23;
	format datatype=dna gap=- matchchar=. ;
matrix

Species1  ATAGAGAGAGAGAGAGAGAGATA
Species2  CTCTCTCTCTCTCTCTGACTCTC
Species3  GAGAGAGCTCTAGAGAGATAGAG
Species4  ACTCTCTCTCTCTCTCTCGCTCT
;
end;
BEGIN PAUP;
	set nowarnreset 
	autoclose;
        nj;
    pscores;
[This block will estimate a maximum parsimony tree assuming equal weights and save the results of the search to a log file.]
  quit warntsave=no;
end;
