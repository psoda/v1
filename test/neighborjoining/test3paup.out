## NJ Completed Successfully
score 60.000000
## PScores Completed Successfully

Done. (at "test3paup.nex" Line 21:3--21:20)
